2022-09-01 23:58:38.920 +02:00 [INF] Now listening on: https://localhost:7145
2022-09-01 23:58:38.946 +02:00 [INF] Now listening on: http://localhost:5145
2022-09-01 23:58:38.949 +02:00 [INF] Application started. Press Ctrl+C to shut down.
2022-09-01 23:58:38.949 +02:00 [INF] Hosting environment: Development
2022-09-01 23:58:38.949 +02:00 [INF] Content root path: D:\repos\BookStoreApp\BookStore.API\
2022-09-01 23:58:47.347 +02:00 [INF] Made call to Weather Endpoint
2022-09-01 23:58:47.348 +02:00 [ERR] Fatal error occurred
System.Exception: this is out logging test exception
   at BookStore.API.Controllers.WeatherForecastController.Get() in D:\repos\BookStoreApp\BookStore.API\Controllers\WeatherForecastController.cs:line 28
2022-09-01 23:58:47.431 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.Exception: this is out logging test exception
   at BookStore.API.Controllers.WeatherForecastController.Get() in D:\repos\BookStoreApp\BookStore.API\Controllers\WeatherForecastController.cs:line 28
   at lambda_method2(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2022-09-01 23:59:34.825 +02:00 [INF] Application is shutting down...
